Gems/ScriptCanvas/Code/Editor/Nodes/NodeCreateUtils.cpp

```diff
@@ -237,6 +253,52 @@ namespace ScriptCanvasEditor::Nodes
         return nodeIdPair;
     }
 
+    NodeIdPair CreateDataDrivenNode(const AZStd::any& nodeData, const AZ::Crc32& nodeLexicalId, const ScriptCanvas::ScriptCanvasId& scriptCanvasId)
+    {
+        return CreateSmallOperatorNode(AZStd::any_cast<SmallOperatorCreationData>(nodeData), nodeLexicalId, scriptCanvasId);
+    }
+
+    NodeIdPair CreateSmallOperatorNode(const SmallOperatorCreationData& nodeData, const AZ::Crc32& nodeLexicalId, const ScriptCanvas::ScriptCanvasId& scriptCanvasId)
+    {
+        ScriptCanvas::Node* node = aznew ScriptCanvas::Node();
+        node->SetNodeName(nodeData.m_title);
+        node->SetNodeToolTip(nodeData.m_toolTip);
+        node->SetNodenodeLexicalId(nodeLexicalId);
+
+        ScriptCanvas::DynamicDataSlotConfiguration inputPin;
+        inputPin.m_name = " ";
+        inputPin.m_toolTip = "Input";
+        inputPin.m_canHaveInputField = false;
+        inputPin.SetConnectionType(ScriptCanvas::ConnectionType::Input);
+        inputPin.m_displayType = nodeData.m_dataType;
+
+        ScriptCanvas::DynamicDataSlotConfiguration outputPin;
+        outputPin.m_name = " ";
+        outputPin.m_toolTip = "Output";
+        outputPin.SetConnectionType(ScriptCanvas::ConnectionType::Output);
+        outputPin.m_displayType = nodeData.m_dataType; 
+
+        AZ::Entity* nodeEntity{ aznew AZ::Entity };
+        nodeEntity->Init();
+        nodeEntity->SetName(node->GetNodeName());
+        nodeEntity->AddComponent(node);
+        ScriptCanvas::GraphRequestBus::Event(scriptCanvasId, &ScriptCanvas::GraphRequests::AddNode, nodeEntity->GetId());
+
+        node->AddSlot(inputPin, true);
+        node->AddSlot(outputPin, true);
+
+        node->SetNodeStyle(GraphCanvas::Styling::Elements::Small);
```

Comment:
Pipe this request through the DataDrivenNode pipeline, and then remove references to any SmallOperatorNode classes, names. Just use the data-driven pipeline that you have provided.